libs/partners/mongodb/langchain_mongodb/chat_message_histories.py

```diff
@@ -97,6 +100,7 @@ def __init__(
         self.collection_name = collection_name
         self.session_id_key = session_id_key
         self.history_key = history_key
+        self.doc_limit = doc_limit
```

Comment:
because developer or the app will not have the access to number of messages in chat session at any point of time. But developer can fix the max messages he/she wants in their application. I have changed the parameter name to max_messages as per @eyurtsev's suggestion.

I hope it explains. Please let me know if you have any more suggestions Thanks!